driver/normalizer/normalizer.go

```diff
@@ -220,8 +249,28 @@ func funcDefMap(typ string, returns bool, other Obj) Mapping {
 				UASTType(uast.Alias{}, Obj{
 					"Name": Var("name"),
 					"Node": UASTType(uast.Function{}, Obj{
-						"Body": Var("body"),
 						"Type": UASTType(uast.FunctionType{}, dstType),
+						// If the function was defined with an arrow expression, we will
+						// generate a uast:Block with a since csharp:Return node containing
```

Comment:
Not sure I can understand this sentence `generate a uast:Block with a ...? since csharp:Return node containing`

Reply:
Thanks! Fixed the typo in the latest commit.